src/Containers/NavBar.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable max-len */
```

Comment:
With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.